src/daemon/http/auth.rs

```diff
@@ -33,7 +33,7 @@ fn build_auth_redirect_location(user: LoggedInUser) -> Result<String, Error> {
     let attributes = b64_encode_attributes_with_mapped_error(&user.attributes)?;
 
     Ok(format!(
-        "/index.html#/login?token={}&id={}&attributes={}",
+        "/ui/login?token={}&id={}&attributes={}",
```

Comment:
We should probably extract this common prefix to a constant, though this issue existed prior to this PR.

Reply:
yeah, I prefer to leave this for now.. we should probably evaluate whether to use a web framework (axum maybe, is that the one you mentioned? @marlonbaeten ) rather than doing our own home grown routing at some point